lib/ui/workspace_registeration_screen/create_workspace/workspace_steps/workspace_step2/create_workspace2_viewmodel.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/cupertino.dart';
+import 'package:stacked/stacked.dart';
+
+class CreateWorkspaceViewModel2 extends BaseViewModel {
+  final ScrollController controllerOne = ScrollController();
+  String _signInPgNum = "Stage 1 of 3 ";
```

Comment:
Same here all constants text in the app can move away and use directly in the UI, since it's not dynamic, we are not changing any thing